spec/features/sign_up_chitter_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'Sign up for Chitter' do
+  scenario 'Sign up as new user' do
+    sign_up
+    expect(page).to have_content("Welcome to my Chitter!")
+    users = User.all
+    expect(users.first.username).to eq "Shaun"
```

Comment:
Should check the feature as a user experiences it, without expecting a result from a behind the curtain method